src/hipscat/catalog/catalog_parameters.py

```diff
@@ -0,0 +1,47 @@
+"""Conatiner class for properties of a catalog that is being created"""
+
+import os
+
+
+class CatalogParameters:
+    """Container class for catalog properties"""
+
+    def __init__(
+        self,
+        catalog_name=None,
+        input_paths=None,
```

Comment:
nit: might be nice to include a type hint here. Unless everyone is already going to know that this is meant to be a list of strings. i.e. `input_paths: List[str] = None`.